### PR TITLE
rest pages amended

### PR DESCRIPTION
**Pull request by CharlineNoga**
Compare page: typos
Job Openings: change of style and wording for Title and subtitle (along the other pages) + Typos
TO DO: - the job opening page needs to call the step, NOT the profession! - the application link

### CODE REVIEW

File: app/views/professions/compare.html.erb [hunk 1]
```diff
@@ -1,25 +1,25 @@
 <h3>How does this sound?</h3>
 <div class="wrapper">
     <div class="first">
-      <a href="/professions/<%= Profession.where(track: 'Product Management').ids[0] %>" class="btn btn-info btn-lg btn-block">Product Mangager</a>
+      <a href="/professions/<%= Profession.where(track: 'Product Management').ids[0] %>" class="btn btn-info btn-lg btn-block">Product Manager</a>
       <div id="D">
-      <h5>Connection between product developement and marketing</h5>
+      <h5>Connection between Product Developement and Marketing</h5>
```
Comment (dwiehoff) [inline app/views/professions/compare.html.erb]: `Developement` >> `Development`

File: app/views/professions/compare.html.erb [hunk 2]
```diff
@@ -1,25 +1,25 @@
...
       </div>
       <div id="K">
       <h4>Knowledge</h4>
       <ul>
-      <li>Bachelor degree or Certificate</li>
-      <li>Product management techniques</li>
-      <li>Lean methodes</li>
+      <li>Bachelor Degree or Certificate</li>
+      <li>Product Management Techniques</li>
+      <li>Lean Methodes</li>
```
Comment (dwiehoff) [inline app/views/professions/compare.html.erb]: `Methodes` >> `Methods`

File: app/views/professions/compare.html.erb (hunk 1 above)
Comment (CharlineNoga) [reply app/views/professions/compare.html.erb]: done

File: app/views/professions/compare.html.erb (hunk 2 above)
Comment (CharlineNoga) [reply app/views/professions/compare.html.erb]: done

**Pull request merged**